dashboard/run_data_collection.py

```diff
@@ -47,6 +47,7 @@ class DataCollector:
 
     utc_time_format = "%m/%d/%y-%H:%M:%S"
     utc_time_gap = datetime.now() - datetime.utcnow()
+    total_start_time = "09/25/18-12:00:00"
```

Comment:
why this date? Doesn't our allocation start from beginning of june?

Reply:
crop out the zeros